xhgui/scripts/settings.ddev.xhgui.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+#ddev-generated
+
+if (file_exists("/mnt/ddev_config/xhgui/collector/xhgui.collector.php")) {
+ require_once "/mnt/ddev_config/xhgui/collector/xhgui.collector.php";
+}
```

Comment:
```suggestion
if (getenv('IS_DDEV_PROJECT') == 'true' && getenv('NO_XHGUI') !== '1') {
    if (file_exists($xhgui = '/mnt/ddev_config/xhgui/collector/xhgui.collector.php')) {
        require_once $xhgui;
    }
}
```

Reply:
Could you explain why this is needed/improved over the current implementation?

Reply:
`getenv('IS_DDEV_PROJECT')` to try to include the file only in DDEV environment.

The `getenv('NO_XHGUI')` is to disable xhgui via ENV var. Name can be changed ...

If the file is included, phpunit (v9.6) does not work.

With that change a can add this to `.phpunit.dist.xml` and everything works.

```xml
    <php>
        <env name="NO_XHGUI" value="1"/>
    </php>
```

Reply:
_If_ we included `getenv('NO_XHGUI')`, we should also add required phpunit configuration in the documentation.

This is outside the scope of this PR as all the other framework guides will also need to be updated.

Reply:
No need to update the other guides if its working for them. The documentation should only mention that it can be used to disable xhgui.